project_definition/views/templates.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="project_term" model="ir.ui.view">
+            <field name="name">project.project.term.form</field>
+            <field name="model">project.project</field>
+            <field name="inherit_id" ref="project.edit_project" />
+            <field name="arch" type="xml">
+                <xpath expr="/form/sheet/notebook/page[@string='Other Info']" position="after">
+                    <page string="Definitions">
+                        <field name="term_ids" />
+                    </page>
+                </xpath>
+            </field>
+        </record>
+
+        <record id="view_definition_tree" model="ir.ui.view">
+            <field name="name">document.definition.tree</field>
+            <field name="model">document.page</field>
+            <field name="priority">100</field>
+            <field name="arch" type="xml">
+                <tree string="Definitions" editable="top">
+                    <field name="name"/>
+                    <field name="content"/>
+                    <field name="type" readonly="1"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="view_definition_form" model="ir.ui.view">
+            <field name="name">document.definition.form</field>
+            <field name="model">document.page</field>
+            <field name="priority">100</field>
+            <field name="arch" type="xml">
+                <form string="Definitions">
+                    <sheet>
+                        <h1>
+                            <field name="name"/>
+                        </h1>
+                        <field name="content"/>
+                        <field name="type" readonly="1"/>
+                    </sheet>
+                </form>
+            </field>
+        </record>
+
+        <!-- filters to group by alphabetical order, and search by term or in
+         definition -->
+
+        <record id="view_definition_filter" model="ir.ui.view">
+            <field name="name">document.definition.search</field>
+            <field name="model">document.page</field>
+            <field name="arch" type="xml">
+                <search string="Definitions">
+                    <field name="name" string="Search Term" filter_domain="[('name','ilike',self)]"/>
```

Comment:
any reason to add a filter domain which does the same as the [default](https://github.com/OCA/OCB/blob/8.0/addons/web/static/src/js/search.js#L1424)?

Reply:
to give it another label. "Search Term"

Reply:
I talk about `filter_domain`